README.md

```diff
@@ -1,80 +1,62 @@
-# react-native-activity-recognition
+# @progstream/react-native-activity-recognition
+
+## Motivation
+
+This is a fork of [react-native-activity-recognition](https://github.com/Aminoid/react-native-activity-recognition) which is unmaintained now. We have updated it to support react-native@0.60+ and fixed some old dependencies.
 
 [![npm version][npm shield]][npm url]
 
 React Native wrapper for the [Android Activity Recognition API][1] and [CMMotionActivity][3]. It attempts to determine the user activity such as
 driving, walking, running and cycling. Possible detected activities for android are [listed here][2] and for iOS are [listed here][3].<br/>
-Updated January 7th and tested with react-native v0.57.5
 
 [1]: https://developers.google.com/android/reference/com/google/android/gms/location/ActivityRecognition
 [2]: https://developers.google.com/android/reference/com/google/android/gms/location/DetectedActivity
 [3]: https://developer.apple.com/reference/coremotion/cmmotionactivity
 [4]: https://facebook.github.io/react-native/docs/linking-libraries-ios.html#manual-linking
```

Comment:
do we really need reference about manual linking? because we don't need it in RN 0.60+

Reply:
Done